reverse_linkedlist/jade_solution.py

```diff
@@ -0,0 +1,41 @@
+def get_data_list(data):
+  data_list = []
+  data_list.append(one['data'])
+  inner_dict = one['next']
+  while inner_dict is not None:    
+  #     inner_dict = one['next']
+      element_data = inner_dict['data']
+      data_list.append(element_data)
```

Comment:
You can combine this and the above line if you want

```suggestion
      data_list.append(inner_dict['data'])
```